gpt_engineer/chat_to_files.py

```diff
@@ -40,3 +41,26 @@ def to_files(chat, workspace):
     files = parse_chat(chat)
     for file_name, file_content in files:
         workspace[file_name] = file_content
+
+
+# Get code content from a code list
+def getCodeStrings(input):
+    filesPaths = input["file_list.txt"].split("\n")
```

Comment:
This looks too cumbersome imo.

We should be able to generate this list of filepaths from the project directory to make it nice for the user!

Could, for example, print the content of the project directory, and ask the user 1. which folders to include and 2. which file extensions to include. And then this is stored to memory.

Reply:
Yes thank you for stating the obvious.

Reply:
If only there were some gpt-engineer functionality that could do this for us. ;)